### PR TITLE
[FEAT] CM-17: implement search doctors filters

### PR DESCRIPTION
**Pull request by alaeddinewannassi**

### CODE REVIEW

File: src/main/java/com/nexym/clinic/config/security/WebSecurityConfig.java [hunk 1]
```diff
@@ -75,6 +75,7 @@ public SecurityFilterChain filterChain(HttpSecurity httpSecurity) throws Excepti
                 // make sure we use stateless session; session won't be used to
                 // store user's state.
                 .exceptionHandling().authenticationEntryPoint(jwtAuthenticationEntryPoint).and()
+                .cors().and()
```
Comment (alaeddinewannassi) [inline src/main/java/com/nexym/clinic/config/security/WebSecurityConfig.java:78]: why adding this change ?

File: src/test/java/com/nexym/clinic/domain/doctor/DoctorServiceTest.java [hunk 2]
```diff
@@ -78,7 +82,7 @@ void should_register_doctor_missing_required_attribute_fail() {
 
     @Test
     void should_search_doctors_success() {
-        var doctorList = doctorService.getDoctorList(0, 10);
+        var doctorList = doctorService.getDoctorList(doctorWsMapper.mapToEnum("TODAY"), 1L, 0, 10);
```
Comment (alaeddinewannassi) [inline src/test/java/com/nexym/clinic/domain/doctor/DoctorServiceTest.java:85]: use AvailableWithinNext.TODAY without using the mapper

File: src/test/java/com/nexym/clinic/domain/doctor/DoctorServiceTest.java [hunk 3]
```diff
@@ -26,6 +27,9 @@ class DoctorServiceTest {
     @Autowired
     private DoctorService doctorService;
 
+    @Autowired
+    private DoctorWsMapper doctorWsMapper;
```
Comment (alaeddinewannassi) [inline src/test/java/com/nexym/clinic/domain/doctor/DoctorServiceTest.java:31]: unuseful import

**Comment by alaeddinewannassi** [comment]
add tests in order to verify that filters work well

File: src/main/java/com/nexym/clinic/config/security/WebSecurityConfig.java (hunk 1 above)
Comment (WnOussama) [reply src/main/java/com/nexym/clinic/config/security/WebSecurityConfig.java:78]: to active the cors bean that I added

**Pull request merged**